packages/react-components/src/components/UnsupportedBrowser.tsx

```diff
@@ -38,5 +39,21 @@ export interface UnsupportedBrowserProps {
  */
 export const UnsupportedBrowser = (props: UnsupportedBrowserProps): JSX.Element => {
   const { onTroubleshootingClick, strings } = props;
-  return <UnsupportedEnvironment onTroubleshootingClick={onTroubleshootingClick} strings={strings} />;
+  const locale = useLocale();
+  return (
+    <UnsupportedEnvironment
+      onTroubleshootingClick={onTroubleshootingClick}
+      strings={strings ? strings : unsupportedBrowserstringsTrampoline(locale)}
```

Comment:
to allow Contoso developer to only override one string, we typically merge strings instead:
```tsx
const localeStrings = unsupportedBrowserstringsTrampoline(locale);
const strings = { ...localeStrings, ...props.strings };
```